src/Microsoft.VisualStudio.InteractiveWindow/Output/OutputClassifierProvider.cs

```diff
@@ -99,6 +99,7 @@ internal sealed class ErrorOutput : ClassificationFormatDefinition
 
                 public ErrorOutput()
                 {
+                    this.DisplayName = InteractiveWindowResources.ErrorOutputFormatDefinitionDisplayName;
```

Comment:
Yes, the are still class level attributes: `[ClassificationType(ClassificationTypeNames = Name)]`, `[Name(Name)]`